pandapower/opf/PpPmInterface/src/pm_models/run_powermodels.jl

```diff
@@ -1,52 +1,54 @@
-# using PowerModels
-# # push!(LOAD_PATH, joinpath(homedir(), "pandapower", "pandapower", "opf"))
-# push!(LOAD_PATH, joinpath(@__DIR__, "pandapower", "pandapower", "opf"))
-# using PP2PM
-
-function run_powermodels(json_path)
-    pm = load_pm_from_json(json_path)
-    model = get_model(pm["pm_model"])
-    #
-    solver = get_solver(pm["pm_solver"], pm["pm_nl_solver"], pm["pm_mip_solver"],
-    pm["pm_log_level"], pm["pm_time_limit"], pm["pm_nl_time_limit"], pm["pm_mip_time_limit"])
-
-    if haskey(pm["branch"]["1"],"c_rating_a")
-        for (key, value) in pm["gen"]
-           # value["pmin"] = 0
-           value["pmax"] *= 0.01
-           value["qmax"] *= 0.01
-           value["qmin"] *= 0.01
-           value["pg"] *= 0.01
-           value["qg"] *= 0.01
-           value["cost"] *= 100
-        end
-
-        for (key, value) in pm["branch"]
-           value["c_rating_a"] *= 0.01
-        end
-
-        for (key, value) in pm["load"]
-           value["pd"] *= 0.01
-           value["qd"] *= 0.01
-        end
-
-        # pm["bus"]["4"]["vmax"] = 1.1
-        # pm["bus"]["4"]["vmin"] = 0.9
-
-        result = PowerModels._run_opf_cl(pm, model, solver,
-                                        setting = Dict("output" => Dict("branch_flows" => true)))
-    else
-        result = PowerModels.run_opf(pm, model, solver,
-                                        setting = Dict("output" => Dict("branch_flows" => true)))
-    end
-    return result
-end
-
-
-# json_path = "C:/Users/fmeier/pandapower/pandapower/test/opf/case5_clm_matfile_va.json"
-# # #@enter run_powermodels(json_path)
-# #
-# result = run_powermodels(json_path)
-# println(result["termination_status"] == LOCALLY_SOLVED)
-# println(isapprox(result["objective"], 17015.5; atol = 1e0))
-# mit eingeschränkter slack spannung: 17082.819507648066
+using PowerModels
+using .PpPmInterface

```

Comment:
the whole idea of PpPmInterface is avoiding multi-modules structure!
run_powermodels.jl supposed to be part of the PpPmInterface, so it is not possible to call a package inside its inner script!